api/ereqs_admin/migrations/0002_auto_20180116_2105.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.5 on 2018-01-16 21:05
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+def forward(apps, schema_editor):
+    Theme = apps.get_model('admin_interface', 'Theme')
+    Theme.objects.create(
+        name="USWDS",
+        active=True,
+        title="OMB Policy Library Editor",
+        title_color="#FFFFFF",
+        title_visible=True,
+        logo="",
+        logo_color="#FFFFFF",
+        logo_visible=False,
+        css_header_background_color="#000000",
+        css_header_text_color="#FFFFFF",
+        css_header_link_color="#FFFFFF",
+        css_header_link_hover_color="#E1F3F8",
+        css_module_background_color="#222222",
+        css_module_text_color="#C2C2C2",
+        css_module_link_color="#C2C2C2",
+        css_module_link_hover_color="#E1F3F8",
+        css_module_rounded_corners=True,
+        css_generic_link_color="#205493",
+        css_generic_link_hover_color="#0071BC",
+        css_save_button_background_color="#3DA4C8",
+        css_save_button_background_hover_color="#3794B5",
+        css_save_button_text_color="#FFFFFF",
+        css_delete_button_background_color="#CD2026",
+        css_delete_button_background_hover_color="#981B1E",
```

Comment:
Several of these colors aren't part of our palette. Can we use our existing palette in place of any of these variations? If we can't combine them, we should extend the palette to include them by adding new color variables to `_colors.scss`.

* E1F3F8
* 222222
* C2C2C2
* 205493
* 0071BC
* 3DA4C8
* 3794B5
* CD2026
* 981B1E